src/xFactor/ParticipantServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package xFactor;
+
+import java.util.ArrayList;
+
+public class ParticipantServiceImpl implements ParticipantService {
+	private StageServiceImpl stageService;
+	public ParticipantServiceImpl() {
+		stageService = new StageServiceImpl();
+	}
+	
+	@Override
+	public void printVotes(Participant participant, Stage stage) {
+		System.out.println("Judges who voted with \"YES\" for " + participant.getName() + ":");
+		
+		if(stageService.countVotes(stage).get(participant) == Competition.getJudgesSize()) {

```

Comment:
stageService.countVotes(stage) is something you use in each case, you call the method every time. This is not good for the performance, extract such things in variables